mcs/build/rules.make

```diff
@@ -344,3 +344,14 @@ MDOC   =$(Q_MDOC) MONO_PATH="$(topdir)/class/lib/$(DEFAULT_PROFILE)$(PLATFORM_PA
 
 Q_MDOC_UP=$(if $(V),,@echo "MDOC-UP [$(PROFILE_DIRECTORY)] $(notdir $(@))";)
 MDOC_UP  =$(Q_MDOC_UP) MONO_PATH="$(topdir)/class/lib/$(DEFAULT_PROFILE)$(PLATFORM_PATH_SEPARATOR)$$MONO_PATH" $(RUNTIME) $(topdir)/class/lib/$(DEFAULT_PROFILE)/mdoc.exe update --delete -o Documentation/en
+
+## Telemetry Stuff
+
+CRASH_REPORTING_UPLOADER_EXE = $(topdir)/class/lib/$(PROFILE)/upload-to-sentry.exe
+# These come from the environment 
+SENTRY_ROOT := $(shell dirname "$(topdir)/../")
+SENTRY_ENV_OPTIONS = MONO_SENTRY_ROOT="$(SENTRY_ROOT)" MONO_SENTRY_OS="$(MONO_SENTRY_OS)" MONO_SENTRY_URL="$(MONO_SENTRY_URL)" 
+
+upload-all: $(CRASH_REPORTING_UPLOADER_EXE)
```

Comment:
`upload-all` is very generic, how about `upload-crashes`.

Reply:
Sounds good